src/SideBar/TestResultTree.ts

```diff
@@ -0,0 +1,71 @@
+import { JestFileResults, JestAssertionResults } from 'jest-editor-support'
+import { TestResult } from '../TestResults'
+
+export class TestResultFile {
+  name: string
+  suites: TestResultSuite[]
+
+  constructor(results: JestFileResults, parsedResults: TestResult[]) {
+    this.name = results.name
+    this.suites = []
+    results.assertionResults.forEach(r => this.parseAssertionResults(r, parsedResults))
+  }
+
+  private parseAssertionResults(results: JestAssertionResults, parsedResults: TestResult[]) {
+    const suite = this.getSuite((<any>results).ancestorTitles, this.suites)
```

Comment:
this crashed when the test file doesn't have the "describe" block outside of the "it" blocks, such as a default CRA app:

results.assertionResults=[{"failureMessages":[],"status":"passed","title":"renders without crashing"}]

> TypeError: Cannot read property 'forEach' of undefined
	at TestResultFile.getSuite (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/TestResultTree.js:28:16)
	at TestResultFile.parseAssertionResults (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/TestResultTree.js:11:28)
	at TestResultFile.results.assertionResults.forEach.r (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/TestResultTree.js:8:52)
	at Array.forEach (<anonymous>)
	at new TestResultFile (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/TestResultTree.js:8:34)
	at JestTreeProvider.loadTestResultsForFile (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/JestTreeProvider.js:35:16)
	at data.testResults.map.r (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/JestTreeProvider.js:30:58)
	at Array.map (<anonymous>)
	at JestTreeProvider.loadTestResults (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/JestTreeProvider.js:30:44)
	at JestTreeProvider.refresh (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/SideBar/JestTreeProvider.js:15:14)
	at JestExt.updateWithData (/Users/vsun/ConnectDotz/External/vscode-jest/out/src/JestExt.js:304:30)